### ISSUE TITLE
During Import of backup ZipFile, links associated with Appointments are not preserved

### ISSUE THREAD

**Issue by arshad4313** [mikeberger/borg_calendar#47]
Description:
-------------------
If you perform an export using the *Import/Export > Export XML…* option, and then import the exported file using *Import/Export > Import Entire Backup Zipfile*, any links associated with an Appointment will not be preserved.

Steps to Reproduce Issue: 
----------------------------------
>	1. Start BORG Calendar 1.8.1
>	2. In the *Month* view, Right-click a date and click *Add New* in order to create a new Appointment.
>	3. In the *Appointment Editor*, fill the Subject field with "Appointment_1" and click *Save*.
>	4. Now select *Appointment_1* from the *Appointment List*.
>	5. Click the *Link file* button in the *Links* section to link a file to *Appointment_1*.
>	6. Select any file; the newly linked file should appear in the *Links* section.
![image](https://cloud.githubusercontent.com/assets/10697341/6005161/2ca24138-aad6-11e4-8de3-f075b41a1a04.png)

>	7. Now perform an export through the *Import/Export > Export XML…* option.
![image](https://cloud.githubusercontent.com/assets/10697341/6005166/32eb5750-aad6-11e4-930f-add9413df2d2.png)

>	8. Next, delete *Appointment_1* by selecting it in the *Appointment List* and clicking the *Delete* button
![image](https://cloud.githubusercontent.com/assets/10697341/6005174/47478e44-aad6-11e4-8aab-20306da6050f.png)

>	9. Import the exported zip file back by using the *Import/Export > Import Entire Backup Zipfile* option
![image](https://cloud.githubusercontent.com/assets/10697341/6005176/4c0aab82-aad6-11e4-8592-a44f44bc82c3.png)

>	10. Go the *Month* view and find *Appointment_1* and open the *Appointment Editor* for it
>	11. Notice that there are no links associated with the Appointment
![image](https://cloud.githubusercontent.com/assets/10697341/6005185/5ba5f0ec-aad6-11e4-8857-c5349b6fa233.png)

Suggested Fix:
----------------------------------
Links should be preserved when an export followed by an import is performed.
		
Test Environments:
----------------------------------
**Primary test environment:**
*BORG - Berger-Organizer 1.8.1
Build Time: 2015-01-17 14:51:28
Java 1.8.0_31
Windows 7 x86-64
Dell Optiplex 745 (Intel Core 2 Duo E6300)*

**Secondary test environment:**
*BORG - Berger-Organizer 1.8.1
Build Time: 2015-01-17 14:51:28
Java 1.7.0_65
Ubuntu 14.04 LTS x86-64
Macbook Pro 13" (Intel Core i5-2415M)*

Metadata:
----------------------------------
*Severity*: Low
*Reproducible*: Yes (always)
*Affected platforms*: All

**Comment by mikeberger** [comment]
maybe you are not saving after adding the link? 
I cannot reproduce this.
Make sure you save after adding the link in step 6.

**Comment by arshad4313** [comment]
I verified it again on both Windows and Ubuntu. I made sure to save after adding the link.

For the sake of saving space, I've only shown the Appointment in the Month view.

1. There is a file linked to the Appointment, as indicated by the @ sign.
![image](https://cloud.githubusercontent.com/assets/10697341/6006275/2d44dec6-aadf-11e4-922f-e171a4f1dd85.png)

2. I then performed the export. And deleted the Appointment.
![image](https://cloud.githubusercontent.com/assets/10697341/6006284/4ec00706-aadf-11e4-8ebe-7f08650baaf2.png)

3. Then performed the import. Note that the file is not linked anymore, as indicated by the absence of the @ sign.
![image](https://cloud.githubusercontent.com/assets/10697341/6006297/6baed8ec-aadf-11e4-99d6-6a27e8d212e5.png)
